### PR TITLE
Feature/EIP1-5687 Return collection reason

### PR DESCRIPTION
**Pull request by matt-wills-valtech**

### CODE REVIEW

**Review by NathanRussellValtech — APPROVED** [review]
LGTM 👍

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 1]
```diff
@@ -1250,6 +1250,11 @@ components:
           $ref: '#/components/schemas/AnonymousSupportingInformationFormat'
         deliveryAddressType:
           $ref: '#/components/schemas/DeliveryAddressType'
+        collectionReason:
+          type: string
+          description: |
+            The reason the elector gave for having to collect their voter ID (if applicable).
```
Comment (vishalgupta-valtech) [inline src/main/resources/openapi/PrintAPIs.yaml:1256]: Minor:  `.... to collect their AED (if applicable)`? Although AED is a voter ID

**Review by vishalgupta-valtech — APPROVED** [review]

File: src/main/resources/openapi/PrintAPIs.yaml (hunk 1 above)
Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:1256]: Also worth updating api spec version

Comment (matt-wills-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml]: Updated. I copied it from another collection reason this spec, which is for AEDs and VACs, but this one isn't 😄

**Pull request merged**